### PR TITLE
fix: made tagline text and button little up

### LINKED ISSUE #6350: Frontpage Header text no longer translatable and moved down
After the recent changes the frontpage header text is no longer translatable. As it now uses two lines of text the "Create Event" button also moved down which makes the area look unnatural.

- [x] Please enable translations and
- [ ] move the text and button up a bit.

Screenshot from 2021-01-13 00-28-09

### PR DESCRIPTION
**Pull request by himanshujaidka**
Fixes #6350 
Screenshot (141)

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by himanshujaidka** [comment]
@iamareebjamal sir, one test is failing

File: app/templates/components/welcome-header.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="content">
-  <h1 class="weight-300 lead">{{t 'The open source event solution for virtual and in-person events.'}}</h1>
+  <h1 class="ui-header">{{t 'The open source event solution for virtual and in-person events.'}}</h1>
```
Comment (sachinchauhan2889) [inline app/templates/components/welcome-header.hbs:2]: I think it must not to be BOLD.

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]

File: app/templates/components/welcome-header.hbs (hunk 1 above)
Comment (himanshujaidka) [reply app/templates/components/welcome-header.hbs:2]: Are you sure? if sure, then I will make changes

**Comment by nlok5923** [comment]
please read the full issue you also need to make the tagline translatable.

**Comment by himanshujaidka** [comment]
> please read the full issue you also need to make the tagline translatable.

sorry, working

**Comment by himanshujaidka** [comment]
@iamareebjamal sir, pls review it

**Comment by mariobehling** [comment]
Please only change position as described in the issue. Do not change anything else.

**Comment by himanshujaidka** [comment]
@mariobehling sir done

**Comment by himanshujaidka** [comment]
@iamareebjamal sir pls tell me if it is right or wrong

File: app/templates/components/welcome-header.hbs (hunk 1 above)
Comment (nlok5923) [inline app/templates/components/welcome-header.hbs:2]: Please only change the position of tagline, just move it little up.

Comment (himanshujaidka) [reply app/templates/components/welcome-header.hbs]: now?

Comment (nlok5923) [reply app/templates/components/welcome-header.hbs]: Ya it's fine now.

Comment (himanshujaidka) [reply app/templates/components/welcome-header.hbs]: Thanks!

**Comment by himanshujaidka** [comment]
@iamareebjamal thanks sir!

**Pull request merged**